airflow/providers/amazon/aws/example_dags/example_sagemaker.py

```diff
@@ -0,0 +1,177 @@
+                    "S3Uri": f"s3://{S3_BUCKET}/config/",
+                    "S3DataDistributionType": "FullyReplicated",
+                }
+            },
+            "CompressionType": "None",
+            "RecordWrapperType": "None",
+        },
+    ],
+    "OutputDataConfig": {
+        "KmsKeyId": "",
+        "S3OutputPath": f"s3://{S3_BUCKET}/training/",
+    },
+    "ResourceConfig": {
+        "InstanceType": "ml.m5.large",
+        "InstanceCount": 1,
+        "VolumeSizeInGB": 5,
+    },
+    "StoppingCondition": {"MaxRuntimeInSeconds": 6000},
+    "RoleArn": f"{ROLE}",
+    "EnableNetworkIsolation": False,
+    "EnableInterContainerTrafficEncryption": False,
+    "EnableManagedSpotTraining": False,
+    "TrainingJobName": TRAINING_JOB_NAME,
+}
+
+SAGEMAKER_CREATE_MODEL_CONFIG = {
+    "ModelName": MODEL_NAME,
+    "Containers": [
+        {
+            "Image": f"{IMAGE_URI}",
+            "Mode": "SingleModel",
+            "ModelDataUrl": f"s3://{S3_BUCKET}/training/{TRAINING_JOB_NAME}/output/model.tar.gz",
+        }
+    ],
+    "ExecutionRoleArn": f"{ROLE}",
+    "EnableNetworkIsolation": False,
+}
+
+SAGEMAKER_INFERENCE_CONFIG = {
+    "TransformJobName": "sample_transform_job",
+    "ModelName": MODEL_NAME,
+    "TransformInput": {
+        "DataSource": {
+            "S3DataSource": {
+                "S3DataType": "S3Prefix",
+                "S3Uri": f"s3://{S3_BUCKET}/config/config_date.yml",
+            }
+        },
+        "ContentType": "application/x-yaml",
+        "CompressionType": "None",
+        "SplitType": "None",
+    },
+    "TransformOutput": {"S3OutputPath": f"s3://{S3_BUCKET}/inferencing/output/"},
+    "TransformResources": {"InstanceType": "ml.m5.large", "InstanceCount": 1},
+}
+
+# [START howto_operator_sagemaker]
+with DAG(
+    "sample_sagemaker_dag",
+    schedule_interval=None,
+    start_date=datetime(2022, 2, 21),
+    catchup=False,
+) as dag:
+    sagemaker_processing_task = SageMakerProcessingOperator(
+        config=SAGEMAKER_PROCESSING_JOB_CONFIG,
+        aws_conn_id="aws_default",
+        task_id="sagemaker_preprocessing_task",
+    )
+
+    training_task = SageMakerTrainingOperator(
+        config=SAGEMAKER_TRAINING_JOB_CONFIG, aws_conn_id="aws_default", task_id="sagemaker_training_task"
+    )
+
+    model_create_task = SageMakerModelOperator(
+        config=SAGEMAKER_CREATE_MODEL_CONFIG, aws_conn_id="aws_default", task_id="sagemaker_create_model_task"
+    )
+
+    inference_task = SageMakerTransformOperator(
+        config=SAGEMAKER_INFERENCE_CONFIG, aws_conn_id="aws_default", task_id="sagemaker_inference_task"
+    )
+
+    model_delete_task = SageMakerDeleteModelOperator(
+        task_id="sagemaker_delete_model_task", config={'ModelName': MODEL_NAME}, aws_conn_id="aws_default"
+    )
+
+    sagemaker_processing_task >> training_task >> model_create_task >> inference_task >> model_delete_task
+    # [END howto_operator_sagemaker]
```

Comment:
Curious if `./breeze build-docs -- --package-filter apache-airflow-providers-amazon` passes locally.  Specifically, the indent level on the START and END tags here are different.  Not sure if that is an issue or not.

Reply:
> Curious if `./breeze build-docs -- --package-filter apache-airflow-providers-amazon` passes locally. Specifically, the indent level on the START and END tags here are different. Not sure if that is an issue or not.

Thanks a lot for review.! Yes the check passed locally for me :)